### DISCUSSION TITLE
Preview modale content misalignement

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#6193] [Ideas & Feature Requests]
- Nova Version: 4.32.11

### Description:
The previewing Typo3 and the content of the modal are not aligned properly.
![Screenshot 2024-02-02 at 4 35 42 PM](https://github.com/laravel/nova-issues/assets/17392251/ce720bee-179f-4724-934d-0882a5e5dd84)

This seems to be related to a negative margin on the modal body.
![Screenshot 2024-02-02 at 4 35 36 PM](https://github.com/laravel/nova-issues/assets/17392251/0d3beda0-4f8b-457f-aba6-9d27b4bc8fc3)

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
